src/utils/config.ts

```diff
@@ -47,14 +49,44 @@ export const ContactPersonParams = t.intersection([
 ]);
 export type ContactPersonParams = t.TypeOf<typeof ContactPersonParams>;
 
-const SpidLogsParams = t.union([
+const SpidLogsStorageAzureStorage = t.interface({
+  SPID_LOGS_STORAGE_CONNECTION_STRING: NonEmptyString,
+  SPID_LOGS_STORAGE_CONTAINER_NAME: NonEmptyString,
+  SPID_LOGS_STORAGE_KIND: withDefault(
+    t.literal("azurestorage"),
+    // for backward compatibility, we assume no kind means azure storage
+    "azurestorage"
+  )
+});
+
+const SpidLogsStorageAwsS3 = t.intersection([
   t.interface({
-    ENABLE_SPID_ACCESS_LOGS: t.literal(true),
+    AWS_ACCESS_KEY_ID: NonEmptyString,
+    AWS_SECRET_ACCESS_KEY: NonEmptyString,
```

Comment:
```suggestion
```

These variables should be removed.
1. because the code does not make explicit reference to them.
2. because in some AWS environment (e.g. Fargate) they do not exists, since credentials are provided with a different method